src/app/page.tsx

```diff
@@ -1,7 +1,317 @@
+import bannerImage from "../../public/banner-image.png";
+import Image from "next/image";
+import ContentContainer from "../../components/ui/ContentContainer";
+import Button from "../../components/ui/Button";
+
 export default function Home() {
   return (
-    <main>
-      <h1>Hello, World!</h1>
+    <main className="grid grid-cols-1 md:grid-cols-10 bg-neutral-white">
+      {/* banner block */}
+      <div className="bg-neutral-white col-span-1 md:col-span-6 aspect-w-16 aspect-h-9 relative">
+        <Image 
+          width={1500}
+          height={850}
+          src={bannerImage}
+          alt="banner-image.png"
```

Comment:
Please adjust all alternative text on this page to be more descriptive. 

For more info regarding accessible descriptions, please see https://accessibility.huit.harvard.edu/describe-content-images